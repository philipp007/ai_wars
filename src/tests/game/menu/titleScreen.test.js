import React from 'react';
import TitleScreen from '../../../game/menu/titleScreen';
import renderer from 'react-test-renderer';

test('TitleScreen should be rendered', () => { 
    const titleScreen = renderer.create(
        <TitleScreen key='1' name="Test" selected={ false } />
    );

    let tree = titleScreen.toJSON();
    expect(tree).toMatchSnapshot();
});