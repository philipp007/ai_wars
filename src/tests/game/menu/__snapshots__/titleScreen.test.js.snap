// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleScreen should be rendered 1`] = `
<div>
  <span
    className="centerScreen"
    style={
      Object {
        "color": "green",
        "fontSize": "80px",
        "top": "5%",
      }
    }
  >
    AI Wars
  </span>
</div>
`;
